apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -67,11 +68,17 @@ export const StrapiHomepageCodec = t.strict({
           }),
         }),
       ]),
-      productsShowcase: t.strict({
+      ecosystem: t.strict({
         title: t.string,
+        productsTabName: t.string,
         products: t.strict({
           data: t.array(ProductCodec),
         }),
+        solutionsTabName: t.string,
+        solutions: t.strict({
+          data: t.array(SolutionCodec),
+        }),
+        solutionsCta: t.union([NullToUndefinedCodec, CallToActionCodec]),
       }),
```

Comment:
I think you should manage the eventuality of a null value of the ecosystem object coming from Strapi
```suggestion
      ecosystem: t.union([
        NullToUndefinedCodec,
        t.strict({
          title: t.string,
          productsTabName: t.string,
          products: t.strict({
            data: t.array(ProductCodec),
          }),
          solutionsTabName: t.string,
          solutions: t.strict({
            data: t.array(SolutionCodec),
          }),
          solutionsCta: t.union([NullToUndefinedCodec, CallToActionCodec]),
        }),
      ]),
```